frontend/src/lib/components/report/ChartElement.svelte

```diff
@@ -32,7 +32,7 @@
 			{chart}
 			data={JSON.parse(data)}
 			width={650}
-			height={300}
+			height={chart.type == ChartType.RADAR ? 600 : 300}
```

Comment:
Kinda weird to have it hard-coded like this but probably fine for now...

Reply:
Yeah sizing across the board for charts is wonky we should standardize it somehow but it's surprisingly tricky.